packages/spector/src/actions/helper.ts

```diff
@@ -1,116 +1,76 @@
-import { HttpMethod, ServiceRequestFile } from "@typespec/spec-api";
-import axios, { AxiosRequestConfig, AxiosResponse } from "axios";
-import FormData from "form-data";
-
-// Don't throw for errors as some scenarios are expected to return 4xx and 5xx errors
-axios.defaults.validateStatus = () => true;
+import { HttpMethod } from "@typespec/spec-api";
+import { MockBody, MockMultipartBody } from "@typespec/spec-api/dist/types.js";
```

Comment:
Should these be exported from the main entry point?

Reply:
yeah they should, also weird its letting me do that, must not be defining `exports` will change that